src/imgproc.rs

```diff
@@ -54,6 +56,25 @@ extern "C" {
         cback_project: *mut CMat,
         ranges: *const *const c_float,
     );
+
+    fn cv_compare_hist(first_image: *const CMat, second_image: *const CMat, method: c_int) -> CResult<c_double>;
+}
+
+/// Possible methods for histogram comparision method
+#[derive(Debug, PartialEq, Clone, Copy, ToPrimitive)]
+pub enum HistogramComparisionMethod {
+    /// HISTCMP_CORREL
+    Corellation = 0,
```

Comment:
s/Corellation/Correlation

Reply:
Let's do it this way: say when you're done with review and I fix it in one commit :)

Reply:
This is it. I thought when I am done with a review, Github will notify you (I need to click a button says something like "you have to address these comments").

Reply:
@nebgnahz I imagine there should be a button `I'm done with review` which sends all comment at once. I think I saw it when I was on Github enterprise. I didn't review anybody on github myself so I just don't know :)